webapp/src/types/gitlab_items.ts

```diff
@@ -56,9 +57,15 @@ export interface Item {
     target: Target;
     approvers: number;
     total_reviewers: number;
+    reviewers: User[];
```

Comment:
Please check if this should be User or you need to define a new type by checking the response from the API?

Reply:
reviewer's data is the same as that of the author's data, so we can use that

Reply:
okay